IDE/Renesas/e2studio/RA6M3G/common/util.h

```diff
@@ -0,0 +1,23 @@
+#ifndef _UTIL_H_
```

Comment:
This file needs the wolfSSL copyright added to top.

Reply:
https://github.com/wolfSSL/wolfssl/blob/91a9117e1b7e7ca316299cef6dfc2243aa127651/IDE/Renesas/e2studio/RA6M3G/common/util.h#L1-L22

Thanks, updated.